src/snowflake/snowpark/context.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2012-2022 Snowflake Computing Inc. All rights reserved.
+#
+
+from snowflake.snowpark.session import Session, _get_active_session
+
+
+def get_active_session() -> Session:
+    """Returns the current active snowpark session. Throws an error when
+    multiple sessions are active or when no sessions are created."""
```

Comment:
Exception in docstring can be like. We use google-style docstring. https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

```python
"""Returns the current active snowpark session. 

Raises:
    SnowparkSessionException: If there are more than one active sessions or no active sessions.
"""
```